dist/Devel-PPPort/parts/inc/misc

```diff
@@ -299,18 +299,8 @@ __UNDEF_NOT_PROVIDED__  PERL_GCC_BRACE_GROUPS_FORBIDDEN
 
 #undef STMT_START
 #undef STMT_END
-#if defined(VOIDFLAGS) && defined(PERL_USE_GCC_BRACE_GROUPS)
-#  define STMT_START    (void)( /* gcc supports ``({ STATEMENTS; })'' */
-#  define STMT_END      )
-#else
-#  if defined(VOIDFLAGS) && (VOIDFLAGS) && (defined(sun) || defined(__sun__)) && !defined(__GNUC__)
-#    define STMT_START  if (1)
-#    define STMT_END    else (void)0
-#  else
-#    define STMT_START  do
-#    define STMT_END    while (0)
-#  endif
-#endif
+#define STMT_START do
+#define STMT_END while (0)
```

Comment:
Wasn't there any good reasons to prefer the other flavor when possible?

Reply:
I thought there were multiple implementations depending on the compiler (besides the brace groups).  Perhaps those went away with C99?